### ISSUE TITLE
Is it possible to choose destinations for dynamic picker?

### ISSUE THREAD

**Issue by abbada25** [greenshot/greenshot#158]
I am honesty not sure if this is a bug or intended. 

If i choose my preferences in this menu --> https://i.imgur.com/fgkGG84.png it does all of them automatically after pressing Prntscrn. I Still want dynamic picker, but without all options. 

Sorry if submiting issue is wrong, i am still pretty new to github :/

**Comment by astrosticks** [comment]
I don't think that this feature exists but I think it would be useful. If "select destination dynamically" is checked, then maybe the checkboxes should still be toggleable, but they show/hide options on the dynamic menu.

**Comment by abbada25** [comment]
Yeah, that is exactly what i wanted.

**Comment by Blacksm1th** [comment]
I agree this would be a very helpful feature.

**Comment by grigsoft** [comment]
+ vote for this feature, and it's not difficult to implement.

**Comment by Lakritzator** [comment]
@grigsoft I'm glad you know how to implement it, I gladly accept your PR!

FYI: Greenshot 2.0 already contains this feature, but that version is long from finished. The feature was very easy to implement in Greenshot 2.0 while I've migrated the configuration to use a completely different UI stack (WPF). The whole migration to dotnet core 3.x also added to making it easier, but it was a long and painful task! Having done the work should pay of as we can more quickly add functionality.

Greenshot 1.3 has an older software stack, and the configuration UI is written in Windows Forms. I'm not sure on what you base you opinion "it's not difficult to implement" maybe it's for an application you made, but for Greenshot 1.3 it's not trivial and on top of that we need translations into many language... which is quite a long process.
